### PR TITLE
Update Oblivion Nouveau Uncut

### PR DESCRIPTION
**Pull request by Sabre142**
Updated note: The author of this mod has recently said this about not cleaning the mod:

> [Do not clean the mod further. It's already been cleaned of all unnecessary ITMs in Tes4Edit (which reports 0 ITMs) and cleaning the mod further could very well break things.](https://forums.nexusmods.com/index.php?/topic/4798455-oblivion-uncut-a-restoration-project/page-175#entry107806288)

### CODE REVIEW

**Comment by sibir-ine** [comment]
Since you've responded to them on Nexus, we can wait for their response before merging this. If the remaining ITMs are indeed intentional, the cleaning data can be replaced with a comment saying not to clean the plugin.

**Comment by Sabre142** [comment]
image

**Review by sibir-ine — APPROVED** [review]

**Pull request merged**
